com.unity.render-pipelines.high-definition/Documentation~/snippets/shader-graph-blocks/smoothness-radial.md

```diff
@@ -0,0 +1,6 @@
+<tr>
+<td>**Smoothness**</td>
+<td>Controls the internal scattering of the light paths and absorption that occurs within the hair fiber. “Allow Radial Smoothness” must be enabled on the hair target to use this property.</td>
```

Comment:
<td>Controls the internal scattering of light paths and the amount of light the hair fiber absorbs. To use this property, enable **Allow Radial Smoothness**.</td>